python/GafferSceneUI/LightEditor.py

```diff
@@ -319,6 +320,57 @@ def __buttonDoubleClick( self, pathListing, event ) :
 
 		return True
 
+	def __keyPress( self, pathListing, event ) :
```

Comment:
Most of this function feels like a multi-selection-aware superset of `__buttonDoubleClick()`. Since the double-clicked item should always be selected, can we just delegate both click and keypress to a single `__editSelectedCells()` method?

Reply:
Yes, much better. Fixed in 05cf74f11693462845f662bb682f6a294d96fd86